modules/S09_firmware_base_version_check.sh

```diff
@@ -508,7 +513,7 @@ S09_firmware_base_version_check() {
 
     if [[ "${THREADED}" -eq 1 ]]; then
       if [[ "${#WAIT_PIDS_S09[@]}" -gt "${MAX_MOD_THREADS}" ]]; then
-        recover_wait_pids "${WAIT_PIDS_S09[@]}"
+        recover_wait_pids WAIT_PIDS_S09
```

Comment:
same here ... could you point me to a doc for better understanding of this mechanism?

Reply:
found it https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameters